From 55bd6688ec22f55f895cde1dfd060132b7e12b28 Mon Sep 17 00:00:00 2001
From: Antonio Terceiro <terceiro@softwarelivre.org>
Date: Fri, 17 Jun 2016 19:00:56 -0300
Subject: [PATCH] lxc-debian: make sure init is installed

init 1.34 is not "Essential" anymore, in order to make it not required
on minimal chroots, docker containers, etc. Because of that we know need
to manually include it on systems that are expected to boot.
---
 templates/lxc-debian.in | 1 +
 1 file changed, 1 insertion(+)

--- a/templates/lxc-debian.in
+++ b/templates/lxc-debian.in
@@ -206,7 +206,16 @@ cleanup()
 
 download_debian()
 {
+    case "$release" in
+      squeeze|wheezy)
+        init=sysvinit
+        ;;
+      *)
+        init=init
+        ;;
+    esac
     packages=\
+$init,\
 ifupdown,\
 locales,\
 libui-dialog-perl,\
